Delete campaign.

jabbarn opened the pull request:
changed tests so it can delete some campaigns, create delete campaign logic.

jabbarn committed:
implement delete campaign logic, change tests

jeisses commented on contracts/force/force.cpp:
```diff
@@ -41,6 +41,20 @@ void force::editcampaign(uint32_t campaign_id, vaccount::vaddress owner, content
                   });
 }
+
+void force::delcampaign(uint32_t campaign_id, vaccount::vaddress owner, content content,vaccount::sig sig) {
```
Let's rename this to `rmcampaign`
As the name for creation is `mkcampaign`, this is more in line with expectations

jeisses replied:
Content is not necessary for deleting a campaign (the parameter is not being used in the function)

jeisses commented on contracts/force/force.cpp:
```diff
@@ -41,6 +41,20 @@ void force::editcampaign(uint32_t campaign_id, vaccount::vaddress owner, content
+  auto camp_itr = camp_tbl.find(campaign_id);
+  eosio::check(camp_itr != camp_tbl.end(), "campaign does not exist");
+
+  delcampaign_params params = {11, campaign_id, content};
```
Let's rename to `rmcampaign_params` as well

jabbarn committed:
remove unused parameters, rename methods, change tests

jeisses committed:
Improve unit tests for rmcampaign

it's good to check that the campaigns are actually removed from the tables on
the blockchain. also a good test to add is to check that someone else than the
campaign owner is not able to execute the action.
